src/shared/migrations/1623001592704-init.ts

```diff
@@ -1,19 +1,19 @@
 import { MigrationInterface, QueryRunner } from 'typeorm';
-import { DATABASE_SCHEMA } from '../entity.model';
+import { MEDIA_SCHEMA } from '../entity.model';
 
 // Initial migration to run whenever the microservie starts
 // It mainly to create scheam & create tables used in the microservie
 
 export class init1623001592704 implements MigrationInterface {
   public async up(queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query(`CREATE SCHEMA IF NOT EXISTS ${DATABASE_SCHEMA}`);
+    await queryRunner.query(`CREATE SCHEMA IF NOT EXISTS ${MEDIA_SCHEMA}`);
     await queryRunner.query(
-      `CREATE TABLE IF NOT EXISTS "${DATABASE_SCHEMA}"."example" ("id" SERIAL NOT NULL, "uuid" uuid NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), "col1" character varying NOT NULL, "col2" character varying NOT NULL, CONSTRAINT "PK_2829ac61eff60fcec60d7274b9e" PRIMARY KEY ("id"))`,
+      `CREATE TABLE IF NOT EXISTS "${MEDIA_SCHEMA}"."media" ("id" SERIAL NOT NULL, "uuid" uuid NOT NULL, "created_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "updated_at" TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(), "uploaded" boolean NOT NULL, "original_url" character varying, "name" character varying NOT NULL, CONSTRAINT "PK_026af23480d8d75860a4abf1a98" PRIMARY KEY ("id"))`,
     );
   }
 
   public async down(queryRunner: QueryRunner): Promise<void> {
-    await queryRunner.query(`DROP SCHEMA IF EXISTS ${DATABASE_SCHEMA} CASCADE`);
-    await queryRunner.query(`DROP TABLE "${DATABASE_SCHEMA}"."example"`);
+    await queryRunner.query(`DROP SCHEMA IF EXISTS ${MEDIA_SCHEMA} CASCADE`);
+    await queryRunner.query(`DROP TABLE IF EXISTS "${MEDIA_SCHEMA}"."media"`);
```

Comment:
The word `media` would be better to get imported as you did in `MEDIA_SCHEMA`.

Reply:
`media` here is the name of the created table in database. The name were generated automatically after the entity name in media.entity.ts file. There is no point of making it variable. If you want to change this name later, you can change it from media.entity.ts file only, then re-generate your migrations.